src/pykeen/nn/combinations.py

```diff
@@ -192,3 +243,67 @@ def __init__(
                 activation=activation,
             ),
         )
+
+
+class SimpleGate(Gate):
+    """A module that implements a gated linear transformation for the combination of entities and literals.
+
+    Implementation based on https://github.com/SmartDataAnalytics/LiteralE/blob/master/model.py Gate class.
+    """
+
+    def __init__(
+        self,
+        entity_embedding_dim: int,
+        literal_embedding_dim: int,
+        activation: HintOrType[nn.Module] = 'sigmoid',
+        activation_kwargs: Optional[Mapping[str, Any]] = None,
+    ) -> None:
+        """Instantiate the :class:`torch.nn.Module`.
+
+        :param entity_embedding_dim: The dimension of the entity representations.
+        :param literal_embedding_dim: The dimension of the literals.
+        :param activation: An optional, pre-instantiated activation module, like :class:`torch.nn.sigmoid`.
+        """
+        super().__init__()
+
+        self.gate_activation = activation_resolver.make(activation, activation_kwargs)
+
+        self.h_layer = nn.Linear(entity_embedding_dim + literal_embedding_dim, entity_embedding_dim)
+
+        self.g_ze_layer = nn.Linear(entity_embedding_dim, entity_embedding_dim)
+
+        self.g_zl_layer = nn.Linear(literal_embedding_dim, entity_embedding_dim)
+
+        self.bias = nn.Parameter(torch.zeros(entity_embedding_dim))
+
+    def forward(self, x: torch.FloatTensor, literal: torch.FloatTensor) -> torch.FloatTensor:
+        """Given the entity and literal representations calucaltes a new embedding that contains information of both."""
+
+        concatenated = torch.cat([x, literal], -1)
+
+        z = self.gate_activation(self.g_ze_layer(x) + self.g_zl_layer(literal) + self.bias)
+
+        h = torch.nn.Tanh()(self.h_layer(concatenated))
```

Comment:
should either save the Tanh instance as a parameter of the model or just use the functional interface